lib/rubocop/cop/rspec/rails/http_status.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+begin
+  require 'rack/utils'
+  AUTOCORRECTS = true
```

Comment:
Ah, there a gotcha here. We want the `AUTOCORRECTS` constant to be defined inside the `RuboCop::Cop::RSpec::Rails::HttpStatus` module and not in global namespace. Not that it’s a problem right now, but if another cop should do the same trick, they’d overwrite each other’s constants.